apps/definition/lib/definition.ex

```diff
@@ -37,6 +37,7 @@ defmodule Definition do
       @impl Definition
       def new(%{} = input) do
         map = for {key, val} <- input, do: {:"#{key}", val}, into: %{}
+        map = Map.put_new(map, :id, UUID.uuid4())
```

Comment:
put_new_lazy might be more performant as it would only generate the UUID if the key isnt already present.

Reply:
Done